### PR TITLE
Code and formatting

### PR DESCRIPTION
**Pull request by ammedmar**

### CODE REVIEW

**Comment by gf2078** [comment]
You removed my comment about it, but there really is something screwy about the arrows in the tikz diagram for Theorem {T: relative MV}. If you're not seeing it, I can send a screen shot of how it's displaying on my computer.

**Comment by ammedmar** [comment]
> You removed my comment about it, but there really is something screwy about the arrows in the tikz diagram for Theorem {T: relative MV}. If you're not seeing it, I can send a screen shot of how it's displaying on my computer.

Yeah, please share the issue. I don't see it.

**Comment by gf2078** [comment]
I sent you a screenshot of the arrow issue by email. After experimenting, it may be a TeXstudio issue.

**Comment by gf2078** [comment]
For the "%consider subsubsection" in Section 7:
This one really should be at the paragraph level. There's a subsubsection in the properties subsection of the products section that's about both naturality and associativity. These naturally go together because they're both situations where there are three maps to M. Within that subsubsection it makes sense to have a naturality paragraph and an associativity paragraph. I don't know if it would work as well to have two subsubsections. I'm not completely opposed, but I don't think it would be as natural an organizational structure.

**Comment by gf2078** [comment]
You wrote: "Additionally, $\pi \td \pi:F_j(x,t) = f(x)$
%	This seems like a typo. Please check"

Good catch: please remove the colon.

**Comment by gf2078** [comment]
I'm good with PR #7 once we settle these things.

**Comment by ammedmar** [comment]
> For the "%consider subsubsection" in Section 7: This one really should be at the paragraph level. There's a subsubsection in the properties subsection of the products section that's about both naturality and associativity. These naturally go together because they're both situations where there are three maps to M. Within that subsubsection it makes sense to have a naturality paragraph and an associativity paragraph. I don't know if it would work as well to have two subsubsections. I'm not completely opposed, but I don't think it would be as natural an organizational structure.

OK, I am just leaving it as is. As I offered, I can redefine the \paragraph cmd to be like what this hack is doing. We can return to this later.

> You wrote: "Additionally, π\tdπ:Fj(x,t)=f(x) % This seems like a typo. Please check"
> Good catch: please remove the colon.

OK!

> I'm good with PR #7 once we settle these things.

Happy to hear, it should be ready now.

**Review by gf2078 — APPROVED** [review]

**Comment by gf2078** [comment]
I approved this just before. Not sure why it's no longer set as approve. Let me try again.

**Review by gf2078 — APPROVED** [review]
